tests/test_datasets/test_loading.py

```diff
@@ -76,13 +81,13 @@ def __init__(self, cache_root: str):
         super().__init__(
             url=...,
             cache_root=cache_root,
-            relative_testing_path='nations/test.txt',
-            relative_training_path='nations/train.txt',
-            relative_validation_path='nations/valid.txt',
+            relative_testing_path=pathlib.PurePath('nations', 'test.txt'),
```

Comment:
I guess this should be `PurePosixPath("nations", "test.txt")`. While windows uses `\` for "real" paths, within tarfiles Posix paths are used again.

Reply:
so this will break on windows as is. I wonder if there's a test case for this one

Reply:
This should now be resolved by converting to a pure path, and using `as_posix` to get the zip file internal relative path.

The tests on Windows should cover this.